docs/contribute/03_style.qmd

```diff
@@ -35,3 +35,59 @@ If you use `nix-shell`, all of these are already setup for you and ready to use,
 ## Docstrings
 We use [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) as our
 standard format for docstrings.
+
+## Documentation, blog, and other prose style
+
+Always capitalize Ibis in prose.
+
+### General points on style and word usage
+
+Avoid the passive voice.  Either Ibis does something or the user does
+  something or a particular backend does something.
+
+Try to avoid using words like "simply", "simple", "obviously", "just", when
+  describing things that Ibis makes easier.  Either the simplicity is
+  self-evident, or it isn't and you are inadvertently insulting the reader.
+
+LLMs do not write well, they write _correctly_.  These are not the same thing.
+
+
+### Text formatting
+
+Wrap long prose strings between 80-90 characters per line. It is very helpful
```

Comment:
perhaps "prose" answers it here, but we don't wrap bullet points, right? that seems to mess up the formatting and I've had some rather long bullet points in recent blogs

Reply:
it shouldn't mess up the formatting -- I can add an example of how to wrap bullets

Reply:
TEACH ME